src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
```

Comment:
Doesn't make sense for me to implement an additional upload endpoint for smaller files. TUS can handle those very well.

We now have the following endpoints:
 - `OPTIONS /myfolder/@upload`: Get info about configuration and supported options
 - `POST /myfolder/@upload`: Request new upload resource for creating a new content object
 - `PATCH /myfolder/@upload/mytusuploaduid`: Upload a file or part
 - `HEAD /myfolder/@upload/mytusuploaduid`: Get current offset of the upload

What we are missing yet are endpoints to replace the file (field) of an existing content object. Something like `POST /myfolder/myfile/@upload/field` and `PATCH /myfolder/myfile/@upload/field`. But that's out of scope for this PR.